src/tools/rados/rados.cc

```diff
@@ -431,15 +431,15 @@ static int do_put(IoCtx& io_ctx, RadosStriper& striper,
     }
     indata.append(buffer::ptr(buffer::create_static(count, buf)));
     if (use_striper) {
-      if (offset == 0)
+      if (obj_offset == 0)
 	ret = striper.write_full(oid, indata);
       else
-	ret = striper.write(oid, indata, count, offset);
+	ret = striper.write(oid, indata, count, obj_offset);
```

Comment:
can we use `offset` instead, and initialize if using `obj_offset` at https://github.com/ceph/ceph/pull/8269/files#diff-c24acfa82f15c38ae9370cab8630a579R411 ?